PUBLISHING.md

```diff
@@ -2,11 +2,11 @@
 
 We use GitHub actions to publish new versions of this repository. To publish, perform the following actions:
 
-1. Checkout the `main` branch and `pull` to ensure your local branch is current — `git checkout main && git pull origin main`
-2. Run `npm version` and select the next version number (being careful to differentiate between releases and release candidates) — `npm version`
-3. Push the resulting commit to the origin's `main` branch — `git push origin main`
-4. Push the resulting tags to origin — `git push origin --tags`
-5. In the GitHub UI, find [the tag you just pushed](https://github.com/Netflix/x-element/tags) and find the "Create release" option.
-6. Add any additional release information (including to check the box if it's a "pre-release", it probably is!).
+1. Ensure your local `main` branch is checked-out and current — `git checkout main && git pull origin main`.
+2. Run `npm version` to view the current version. Then…
```

Comment:
Surprise! The functionality of `npm version` has _changed_ in the latest version. Sort of surprising, but not a huge deal. It _used_ to prompt you for a version after running, now, you have to pass the versioning information up-front. Sorta sensible, but caught me off guard earlier.